packages/shared/src/components/auth/SocialRegistrationForm.tsx

```diff
@@ -60,11 +64,40 @@ export const SocialRegistrationForm = ({
   isLoading,
 }: SocialRegistrationFormProps): ReactElement => {
   const { trackEvent } = useContext(AnalyticsContext);
-  const { user } = useContext(AuthContext);
+  const { user, updateUser } = useContext(AuthContext);
   const [nameHint, setNameHint] = useState<string>(null);
   const [usernameHint, setUsernameHint] = useState<string>(null);
   const [twitterHint, setTwitterHint] = useState<string>(null);
+  const [name, setName] = useState<string>('');
```

Comment:
Since we pass the string literal, I think the type will be inferred.
```suggestion
  const [name, setName] = useState('');
```